lib/pavilion/series.py

```diff
@@ -155,6 +161,38 @@ def _save_series_id(self):
             self._logger.warning("Could not save series id to '%s'",
                                  last_series_fn)
 
+    def _sys_name_series_tracking(self):
```

Comment:
You should be replacing the old way of tracking series for users, not adding a separate way.